README.md

```diff
@@ -1,2 +1,665 @@
+    end
+  end
+end
+```
+#### Sanitize attributes
+
+You can strip all HTML tags from your attributes with the `sanitize` option.
+
+```ruby
+class Book < ActiveRecord::Base
+  include MeiliSearch
+
+  meilisearch :sanitize => true do
+  end
+end
+```
+#### UTF-8 encoding
+
+You can force the UTF-8 encoding of all your attributes using the `force_utf8_encoding` option.
+
+```ruby
+class Book < ActiveRecord::Base
+  include MeiliSearch
+
+  meilisearch :force_utf8_encoding => true do
+  end
+end
+```
+
+### Manual operations
+
+#### Indexing & deletion
+
+You can manually index a record by using the `index!` instance method and remove it by using the `remove_from_index!` instance method
+
+```ruby
+  book = Book.create!(title: 'The Little Prince', author: 'Antoine de Saint-Exupéry')
+  book.index!
+  book.remove_from_index!
+  book.destroy!
+```
+
+To reindex all your records, use the `reindex!` class method:
+
+```ruby
+  Book.reindex!
+
+  # You can also index a subset of your records
+  Book.where('updated_at > ?', 10.minutes.ago).reindex!
+```
+
+To delete all your records, use the `clear_index!` class method
+
+```ruby
+  Book.clear_index!
+```
+
+#### Access the underlying index object
+
+To access the index object and use the meilisearch-ruby index methods, call the `index` class method:
+
+```ruby
+  index = Book.index
+  # index.get_settings, index.number_of_documents
+```
+
+### Development & testing
+#### Exceptions
+
+You can disable exceptions that could be raised while trying to reach MeiliSearch's API by using the `raise_on_failure` option:
+
+```ruby
+class Book < ActiveRecord::Base
+  include MeiliSearch
+
+  # only raise exceptions in development environment
+  meilisearch :raise_on_failure => Rails.env.development? do
+  end
+end
+```
+
+#### Testing
+  ##### Synchronous testing
+  You can force indexing and removing to be synchronous by setting the following option:
+
+  ```ruby
+  class Book < ActiveRecord::Base
+    include MeiliSearch
+
+    meilisearch synchronous: true do
+    end
+  end
+  ```
+  🚨 This is only recommended for testing purposes, the gem will call the `wait_for_pending_update` method that will stop your code execution until the asynchronous task has been processed by MeilSearch.
+
+  ##### Disable auto-indexing & auto-removal
+
+  You can disable auto-indexing and auto-removing setting the following options:
+
+  ```ruby
+  class Book < ActiveRecord::Base
+    include MeiliSearch
+
+    meilisearch auto_index: false, auto_remove: false do
+    end
+  end
+  ```
+
+  You can temporarily disable auto-indexing using the without_auto_index scope:
+
+  ```ruby
+  Book.without_auto_index do
+    1.upto(10000) { Book.create! attributes } # inside this block, auto indexing task will not run.
+  end
+  ```
+
+## 🤖 Compatibility with MeiliSearch
+
+This package only guarantees the compatibility with the [version v0.20.0 of MeiliSearch](https://github.com/meilisearch/MeiliSearch/releases/tag/v0.20.0).
```

Comment:
Can you put this section between `Documentation` and `Installation`. As the README is huge, I'm afraid the users don't see this important section 🙂